internal/wasm/store.go

```diff
@@ -595,10 +595,27 @@ func executeConstExpression(globals []*GlobalInstance, expr *ConstantExpression)
 		case ValueTypeF64:
 			v = api.DecodeF64(g.Val)
 		}
+	case OpcodeRefNull:
+		switch expr.Data[0] {
+		case ValueTypeExternref:
+			v = int64(0) // Extern reference types are opaque 64bit pointer at runtime.
+		case ValueTypeFuncref:
+			// For funcref types, the pointer value will be set by Engines, so
+			// here we set the "invalid function index" (-1) to indicate that this should be null reference.
+			v = GlobalInstanceNullFuncRefValue
+		}
+	case OpcodeRefFunc:
```

Comment:
is extern missing here for a reason? if so, add a comment why?

Reply:
there's not ref.extern opcode!